README.md

```diff
@@ -1,2 +1,171 @@
 # learn-seo
 :mag: Learn search engine optimisation :sparkles:
+
+## Why should I care about Search Engine Optimisation?
+If a tree falls in the woods and no one is there to
+hear it, does it make a sound? If a website isn't
+on the first page of a google search, does how good
+it is even matter? :thinking:
+
+The majority of traffic for sites still comes from organic searches (this means
+people searching using normal search queries).
+
+There is an increase in traffic coming from social
+media and people sharing things, but the majority
+is still coming from search engines. So you want to
+make sure your site shows
+up for the relevant queries, so that it gets optimum
+traffic, and all of the benefits that come with it.
+
+ You can make the best website in the world, but if _no one_ can find it, then it doesn't matter!
+
+And being high up in searches matters! A study indicated that the first result
+from a google search recieves 32.5% of clicks, where the second receives only
+17.6%, that's almost half the traffic, due to only being one position lower!
+[source](https://searchenginewatch.com/sew/study/2276184/no-1-position-in-google-gets-33-of-search-traffic-study)
+
+## What is SEO?
+
+Search engine optimisation is an effort to increase the traffic to a website by
+trying to make sure it appears as high in the search results as possible for
+searches related to certain queries.
+
+Before looking into SEO too much it is useful to have a little understanding
+of the way searches on the web work for which we really recommend [this short
+video on how google searches work](https://www.youtube.com/watch?v=BNHR6IQJGZs).
+[
+![screen shot of a youtube video with the title "How Search Works by Matt Cutts"](https://user-images.githubusercontent.com/21139983/30594806-4808aa00-9d47-11e7-8267-fd8bad7304ad.png)
+](https://www.youtube.com/watch?v=BNHR6IQJGZs).
+
+By understanding the way that searches work, we can try to make sure that our
+website has as high of a chance as possible to be high result in searches
+relevant to our topic. It's still important to remember to keep
+[User Experience](https://en.wikipedia.org/wiki/User_experience_design)
+at the heart of all of our decisions, and make sure we are not
+compromising UX for the sake of SEO just as google themselves advocate:
+
+> Search engine optimization is about putting your site's best foot
+forward when it comes to visibility in search engines, but your
+ultimate consumers are your users, not search engines.
+
+>_[From google's own guide to SEO](https://static.googleusercontent.com/media/www.google.com/en//webmasters/docs/search-engine-optimization-starter-guide.pdf)_
+
+## How do I do it?
+
+A lot of SEO is making a clear site with good quality content that users will
+engage with, but there are some basic tips you can do on your own site which
+will make sure that search engines will show your site in relevant searches.
```

Comment:
there are some basic tips that can help to make sure that search engines show your site in relevant searches.